### ISSUE TITLE
Conversion to TFLite crashes without any Error

### ISSUE THREAD

**Issue by msquigle** [tensorflow/tensorflow#53744]
### System information

-   **Have I written custom code (as opposed to using a stock example script
    provided in TensorFlow)**:
-   **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10, 10.0.19043 Build 19043
-   **TensorFlow installed from (source or binary)**: Binary
-   **TensorFlow version (use command below)**: v2.5.0-rc3-213-ga4dfb8d1a71 2.5.0
-   **Python version**: Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)] on win32
-   **CUDA/cuDNN version**: 
-   **GPU model and memory**: GeForce GTX 1050
-   **Exact command to reproduce**:
```
import tensorflow as tf

model_file = 'mobilenet_v2_1.0_224_frozen_pb'
converter = tf.lite.TFLiteConverter.from_saved_model(model_file)
converter.experimental_new_converter = True
converter.allow_custom_ops = True
tflite_model = converter.convert()

with open(model_file + '.tflite', 'wb') as f:
    f.write(tflite_model)
```
[mobilenet_v2_1.0_224_frozen_pb.zip](https://github.com/tensorflow/tensorflow/files/7858479/mobilenet_v2_1.0_224_frozen_pb.zip)
### Describe the problem
When running this command with the model provided, the script terminates without any error message or conversion to TFLite. The model is created from converting an ONNX model to Saved Model using the onnx_tf library.

Expected Behaviour: The model converts to TFLite or provides an error message for further debugging.

### Source code / logs
2022-01-12 16:02:06.422817: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-01-12 16:02:06.424495: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1258] Device interconnect StreamExecutor with strength 1 edge matrix:
2022-01-12 16:02:06.426841: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1264]
2022-01-12 16:02:21.969931: W tensorflow/compiler/mlir/lite/python/tf_tfl_flatbuffer_helpers.cc:345] Ignored output_format.
2022-01-12 16:02:21.970194: W tensorflow/compiler/mlir/lite/python/tf_tfl_flatbuffer_helpers.cc:348] Ignored drop_control_dependency.
2022-01-12 16:02:21.972195: W tensorflow/compiler/mlir/lite/python/tf_tfl_flatbuffer_helpers.cc:354] Ignored change_concat_input_ranges.
2022-01-12 16:02:21.974709: I tensorflow/cc/saved_model/reader.cc:38] Reading SavedModel from: mobilenet_v2_1.0_224_frozen_pb
2022-01-12 16:02:22.188729: I tensorflow/cc/saved_model/reader.cc:90] Reading meta graph with tags { serve }
2022-01-12 16:02:22.189000: I tensorflow/cc/saved_model/reader.cc:132] Reading SavedModel debug info (if present) from: mobilenet_v2_1.0_224_frozen_pb
2022-01-12 16:02:22.190593: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1258] Device interconnect StreamExecutor with strength 1 edge matrix:
2022-01-12 16:02:22.190790: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1264]
2022-01-12 16:02:23.052605: I tensorflow/cc/saved_model/loader.cc:206] Restoring SavedModel bundle.
2022-01-12 16:02:24.285677: I tensorflow/cc/saved_model/loader.cc:190] Running initialization op on SavedModel bundle at path: mobilenet_v2_1.0_224_frozen_pb
2022-01-12 16:02:25.318756: I tensorflow/cc/saved_model/loader.cc:277] SavedModel load for tags { serve }; Status: success: OK. Took 3344022 microseconds.
2022-01-12 16:02:29.185145: I tensorflow/compiler/mlir/tensorflow/utils/dump_mlir_util.cc:210] disabling MLIR crash reproducer, set env var `MLIR_CRASH_REPRODUCER_DIRECTORY` to enable.
2022-01-12 16:02:31.117089: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1733] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce GTX 1050 computeCapability: 6.1
coreClock: 1.442GHz coreCount: 6 deviceMemorySize: 3.00GiB deviceMemoryBandwidth: 78.32GiB/s
2022-01-12 16:02:31.117492: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1766] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2022-01-12 16:02:31.558830: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1258] Device interconnect StreamExecutor with strength 1 edge matrix:
2022-01-12 16:02:31.559036: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1264]      0
2022-01-12 16:02:31.559895: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1277] 0:   N

The command fully terminates at this line without anything further or debugging.

**Comment by mohantym** [comment]
Hi @msquigle ! I was able to resolve the issue by adding below line in TF 2.7 . Attaching [gist](https://colab.sandbox.google.com/gist/mohantym/db9e3bb16b4e66d55aec2c916610c177/github_53744.ipynb#scrollTo=ltzDBqiYqPaO) for reference. Thank you!

> converter.optimizations = [tf.lite.Optimize.DEFAULT]

**Comment by msquigle** [comment]
Thank you! That also resolves the issue in 2.5!

**Comment by msquigle** [comment]
@mohantym So this partially solves my issue, I am looking to generate a full float TFLite, which adding that line does not allow. Is there a solution that is possible for generating a non-quantized TFLite model?

**Comment by mohantym** [comment]
@msquigle ! The original code is also working with out  "converter.optimizations = [tf.lite.Optimize.DEFAULT]"  snippet. Thank you!

**Comment by mohantym** [comment]
Closing this issue as it is resolved in 2.7. Please reopen this issue if needed. Thank you!
